public/index.html

```diff
@@ -40,4 +40,17 @@
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
   </body>
+  <script>
+    // First we get the viewport height and we multiple it by 1% to get a value for a vh unit
```

Comment:
Presuming this is the intent (is this needed in modern browsers?). Remove other comments and:

```suggestion
    // Fix the vh unit so that navbar etc. doesn't bounce the content around.
```

Reply:
Removed Script